### ISSUE TITLE
Enhancement: Show compiler options when building

### ISSUE THREAD

**Issue by ericbmerritt** [erlware-deprecated/sinan#6]
What steps will reproduce the problem?
1. sinan build

What is the expected output? What do you see instead?

The compiler options are not shown anywhere when building, so I added a log
message to show the options before each application is compiled. This is
needed because if using the compile_args option in _build.cfg, there's no
way to know if it actually works. It's also really useful to know which
flags are being used.

What version of the product are you using? On what operating system?
0.10.0.14

Please provide any additional information below.
Patch forthcoming.

**Comment by efine** [comment]
I find this incredibly useful and use it all the time.

**Comment by ericbmerritt** [comment]
Did you ever send that patch? Sorry to be so derelict on this.

**Comment by efine** [comment]
I did, and you modified it later to use a new option, print_args, to allow a choice in the matter.

I have a fork of the sinan github repository at http://github.com/efine/sinan if you want to look around. With the changes I made there, I use a command line that looks like this:

sinan build tasks:build:compile_args +debug_info:-DDEBUG_BUILD=true task:build:print_args True

The colons are syntax that I added to overcome the Python space parsing problem (in commit 87867b6890beb010721fd13d7c09972f92b96e3a in my fork).
There is also a commit 975344225a4b57f7cede466ac18ebd2e874ab12a to compile all .erl files subdirectories of src as well as those in src.
